cleanup the intro, and clarify how it relates to the reference (#140)
rust-lang-nursery/nomicon: src/README.md

basics where appropriate. You can skip straight to this book if you want;
just know that we won't be explaining everything from the ground up.

This book exists primarily as a high-level companion to [The Reference][ref].

> Where The Reference is essentially a giant exhaustive list of every individual

Reviewer: ```suggestion
Where The Reference intends to develop into a formal specification of the Rust's
```
The reference is not intended as a *list* of bullet points or anything like that and moreover does not cover the standard library.

> component of Rust and what semantics it has, The Rustonomicon serves to describe

Reviewer: ```suggestion
syntax and semantics, The Rustonomicon serves to describe
```

how to put those pieces together, and the issues that you will have in doing so.

> The Rustonomicon is the "integration tests" to The Reference's "unit tests".

Reviewer: ```suggestion
```
Not sure this analogy works... The reference is aimed at being *formal* but should cover everything... tests are not that...

It should be noted that when The Rustonomicon was originally written, The
Reference was in a state of complete disrepair, and so many things that should
have been covered by The Reference were originally only documented here. Since
then, the reference has been revitalized and is properly maintained, although
it is still far from complete. In general, if the two documents disagree, The

> Reference should be assumed to be correct.

Reviewer: ```suggestion
Reference should be assumed to be correct. Note however that The Reference is currently not normative.
```
